src/diffpy/structure/tests/testloadstructure.py

```diff
@@ -41,6 +42,16 @@ def test_cif(self):
         self.assertFalse(isinstance(stru, PDFFitStructure))
         return
 
+    def test_cif_pathlib(self):
```

Comment:
Please change this to test loading of windows path (of str type) which breaks without this PR.
You may need to decorate it with [@unittest.skipIf](https://docs.python.org/3/library/unittest.html#unittest.skipIf) 
so that the test is only run on Windows OS.

BTW, are all unit tests passing with the recent PyCifRW?
If the bug already shows on an existing test, you can do without adding a new one.

Reply:
Thanks for the tip. With PyCifRW 4.4, the current tests fail for Python 3, but pass for Python 2.

For PyCifRW 4.3, the current tests pass for both Python versions.

I have removed the new test in this case as the current test is sufficient.